<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta name="robots" content="all,follow">
  <meta name="googlebot" content="index,follow,snippet,archive">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Poster Schedule</title>
  <meta name="author" content="" />

  
  <meta name="keywords" content="useR, hugo, conference">	
  

  
  <meta name="description" content="The conference for users of R from July 10-13, 2018 held in Brisbane, Australia.">	
  

  <meta name="generator" content="Hugo 0.82.0" />

  <link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel='stylesheet' type='text/css'>

  
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">

  
  <link href="https://user2018.r-project.org/css/animate.css" rel="stylesheet">

  
  
    <link href="https://user2018.r-project.org/css/style.blue.css" rel="stylesheet" id="theme-stylesheet">
  


  
  <link href="https://user2018.r-project.org/css/custom.css" rel="stylesheet">

  
    

  
  <link rel="shortcut icon" href="https://user2018.r-project.org/img/favicon.ico" type="image/x-icon" />
  <link rel="apple-touch-icon" href="https://user2018.r-project.org/img/apple-touch-icon.png" />
  

  <link href="https://user2018.r-project.org/css/owl.carousel.min.css" rel="stylesheet">
  <link href="https://user2018.r-project.org/css/owl.theme.default.min.css" rel="stylesheet">

  <link rel="alternate" href="https://user2018.r-project.org/index.xml" type="application/rss+xml" title="useR! 2018">

  
  <meta property="og:title" content="Poster Schedule" />
  <meta property="og:type" content="website" />
  <meta property="og:url" content="/poster//" />
  <meta property="og:image" content="img/user_au_logo.png" />

</head>


  <body>

    <div id="all">

        <header>

          <div class="navbar-affixed-top" data-spy="affix" data-offset-top="200">

    <div class="navbar navbar-default yamm" role="navigation" id="navbar">

        <div class="container">
            <div class="navbar-header">
                <a class="navbar-brand home" href="https://user2018.r-project.org/">
                    <img src="https://user2018.r-project.org/img/user_au_logo_small.png" alt="Poster Schedule logo" class="hidden-xs hidden-sm">
                    <img src="https://user2018.r-project.org/img/user_au_logo_small.png" alt="Poster Schedule logo" class="visible-xs visible-sm">
                    <span class="sr-only">Poster Schedule - go to homepage</span>
                </a>
                <div class="navbar-buttons">
                    <button type="button" class="navbar-toggle btn-template-main" data-toggle="collapse" data-target="#navigation">
                      <span class="sr-only">Toggle Navigation</span>
                        <i class="fa fa-align-justify"></i>
                    </button>
                </div>
            </div>
            

            <div class="navbar-collapse collapse" id="navigation">
                <ul class="nav navbar-nav navbar-right">
                  
                  <li class="dropdown">
                    
                    <a href="/">Home</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Programme <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                      
                        <li><a href="/scientific/">Scientific Programme</a></li>
                      
                        <li><a href="/schedule/">Talk Schedule</a></li>
                      
                        <li><a href="/lightning/">Lightning Talk Schedule</a></li>
                      
                        <li><a href="/poster/">Poster Schedule</a></li>
                      
                        <li><a href="/presenter/">Info for presenters</a></li>
                      
                        <li><a href="/tutorials/">Tutorials</a></li>
                      
                        <li><a href="/rcurious/">FREE Tutorial</a></li>
                      
                        <li><a href="/datathon/">Datathon</a></li>
                      
                        <li><a href="/social/">Social Programme</a></li>
                      
                    </ul>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Registration <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                      
                        <li><a href="/registration/">Register</a></li>
                      
                        <li><a href="/abstract/">Abstract Submission</a></li>
                      
                        <li><a href="/scholarship/">Scholarship</a></li>
                      
                    </ul>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/code_of_conduct/">Code of Conduct</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/blog/">News</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Travel <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                      
                        <li><a href="/travel/">Travel to/from/around</a></li>
                      
                        <li><a href="/hotels/">Hotels</a></li>
                      
                        <li><a href="/visit/">Things to do</a></li>
                      
                    </ul>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/faq/">FAQ</a>
                    
                  </li>
                  
                  <li class="dropdown">
                    
                    <a href="/contact/">Contact</a>
                    
                  </li>
                  
                </ul>
            </div>
            

            <div class="collapse clearfix" id="search">

                <form class="navbar-form" role="search">
                    <div class="input-group">
                        <input type="text" class="form-control" placeholder="Search">
                        <span class="input-group-btn">

                    <button type="submit" class="btn btn-template-main"><i class="fa fa-search"></i></button>

                </span>
                    </div>
                </form>

            </div>
            

        </div>
    </div>
    

</div>




        </header>

        <div id="heading-breadcrumbs">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <h1>Poster Schedule</h1>
            </div>
        </div>
    </div>
</div>


        <div id="content">
            

            <div class="container">

                <div class="row">

                    <div class="col-md-9">

                        <div>
                          <p>The talks will take place on 11-13 July 2018 (click the interested talk for its abstract). Poster presenters have the option to do a 30 second lightning talk after the first keynote, to advertise their poster.  Information for presenters is <a href="../presenter.html">here</a>.</p>
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<pre><code>&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;36&quot; data-target=&quot;.36collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Shinichi Takayanagi&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;How LINE Corp Use R to Compete in a Data-Driven World&lt;/td&gt;  &lt;td&gt;applications, reproducibility, Practical use of R language in business&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 36collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;LINE is one of the most popular messaging applications in Asia developed by LINE corp.We always collect and utilize the log of user behavior to import our services.In our data science team, R plays an important role in all stages of the data-analysis and science.It ranges from exploratory data analysis and modeling to sharing the results using Shiny with other colleagues.In our poster presentation, we explain how we use R to solve our business problems and share some practical insights for others who want to use R in their business.At first, as our data science team grows, different people develop their own R script to solve similar tasks like data base connection.We started working on our internal R package called &quot;liner&quot; shared through Github Enterprise to solve this problems.Now, multiple members often collaborate in order to improve and fix bugs using this package.We also introduce an overview of our R-related data analysis platform along with some useful OSS tools:- yanagishima: It helps people run query easily developed at LINE corp- Drone: Docker-based continuous integration tool. We use this for automated unit test and deploy &quot;liner&quot; package to users&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;63&quot; data-target=&quot;.63collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Mahmoud Ahmed&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;pcr: an R package for quality assessment, analysis and testing of qPCR data&lt;/td&gt;  &lt;td&gt;bioinformatics, data-access&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 63collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Real-time quantitative PCR (qPCR) is a broadly used technique in the biomedical research. We developed an R package to implement methods for quality assessment, analysis and testing qPCR data for statistical significance. The Double Delta CT and standard curve models were implemented to quantify the relative expression of target genes from CT values in standard qPCR. In addition, calculation of amplification efficiency and curves from serial dilution qPCR experiments are used to assess the data quality. Finally, two-group testing and linear models are used to test for statistical significance among conditions. Using two datasets from qPCR experiments, we applied different quality assessment, analysis and statistical testing in the pcr package and compared the results to the original published articles. The pcr package provides an intuitive and unified interface for its main functions to allow biologist to perform all necessary steps of qPCR analysis and produce graphs in a uniform way.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;69&quot; data-target=&quot;.69collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Miles McBain&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;An R pipeline for creating and hosting collaborative Web VR environments.&lt;/td&gt;  &lt;td&gt;visualisation, space/time&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 69collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;We are continuing the rich history of R as a generative tool for data driven documents by developing a capability to generate Virtual Reality (VR) scenes. A VR scene object is provided that can create, configure, and serve multi-user Web VR scenes. Development so far includes tools that harness the R spatial ecosystem to create textured 3D meshes from contour and raster data. Several other VR primitives are provided, including 360 photo 'portals'.The immediate application for this work has been the creation of environments for the calibration of geo-spatial statistical models by expert elicitation. However, we see the ability to generate and host multi-user scenes, complete with facility for activity logging, as a compelling new platform for collecting and visualising experimental data.Given this work is evolving rapidly, a poster would be the perfect tool to engage attendees in a variety of conversations about the API, the technology stack, and applications. If given a poster slot at useR!2018, we would like to bring some VR hardware to the session, to give attendees a chance to experience some of the virtual environments our tools have generated. &lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;6&quot; data-target=&quot;.6collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Kim Ki-Yeol&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Squamous cell carcinoma analysis with R&lt;/td&gt;  &lt;td&gt;bioinformatics&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 6collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Squamous cell carcinoma (SCC) is the most common histological type of head neck cancer and cervical cancer. Carcinogenesis in these two types of cancers demonstrate similar multistep progression. The purpose of the study is to identify the significant consensus gene modules of these two cancers. We used a publicly available expression dataset for the study. The dataset included head neck cancer (42 cancer samples and 14 normal samples) and cervical cancer (20 cancer samples and 8 normal samples). We used only human papilloma virus 16 positive samples for excluding the bias according to the different types of HPV. We identified consensus modules of two types of cancers and explored the biological functions of each modules by annotation tool. We identified 8 consensus gene modules of head neck cancer and cervical cancer. Each module was well preserved between the two types of cancer. The modules included significant biological functions, including ATP binding and extracellular exosome. Consensus gene module identification is expected to contribute to more personalized management of multiple cancer types.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;95&quot; data-target=&quot;.95collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Erika Siregar&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;AnalevR: An Interactive R-Based Analysis Environment for Utilizing BPS-Statistics Indonesia Data&lt;/td&gt;  &lt;td&gt;visualisation, web app, community/education, networks, big data, analysis, collaboration, cloud computing, analysis as a service, bps, statistics Indonesia, analysis environment, remote data analysis&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 95collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;As Indonesia’s national statistical agency, BPS-Statistics Indonesia produces a massive amount of strategic data every year. However, these data are still underutilized by other parties (governments, researchers, etc.) due to technical limitations and raw data exclusivity and locality. Actually, numerous people outside BPS are capable of conducting analysis but unable to access the data. To increase the data usefulness, we introduce AnalevR, an online R-based analysis environment that allows anyone to perform analyses and create visualization without having to own the original raw data. It uses a notebook-like interface where users can type commands and the output appears below it. BPS provides the data and analysis service (including the R modules) which are held in cloud storage and can be explored via the helper function. Users remotely execute R commands and perform analysis inside the workspaces. A user can create up to 10 workspaces, each representing different sessions. Each saved session preserves the user-defined variables and functions for future use. This breakthrough will raise users’ involvement in employing BPS’ data and increase statistical quality in Indonesia.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;111&quot; data-target=&quot;.111collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Yan Holtz&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;From Data to Viz&lt;/td&gt;  &lt;td&gt;visualisation, community/education&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 111collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Selecting the right graphic type is a common task for a data scientist. On a daily basis, an R user deals with a data frame and must decide what visualization is the most appropriate to represent it.The task is not easy. The data scientist must:-	Know the broad spectrum of visualization types-	Figure out which dataviz is doable given the dataset-	Try several (or all) of them-	Find the code to create the charts-	Avoid the common caveats associated with the selected optionData-to-viz.com is a new website that comes to meet these needs. It displays an interactive decision tree. The user describe their dataset, what leads them to a set of appropriate graphic types. A description is provided for each, explaining its pros and cons. Links to the R and the Python graph galleries are provided, which allows to get the corresponding code in seconds.The complete decision tree is also available in a static version through a poster. The project has not been released yet due to its potential announcement at the useR conference.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;148&quot; data-target=&quot;.148collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Goknur Giner&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Pathway-VisualiseR&lt;/td&gt;  &lt;td&gt;visualisation, web app, community/education, bioinformatics, networks, Bioconductor&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 148collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Statistical modelling of any genomic research produces sets of genes and individual biomarkers which require investigation. Further  exploration of those set of biomarkers is a prominent step towards  discovering the source of a biological problem. Furthermore,  understanding the collective behaviour of genes has been shown to provide valuable insights into the triggers of many human diseases. We  have developed an RShiny application that provides an interface for  researchers, enabling them to discover the interaction between their  genes and biological pathways. This application allow users to inquire  into details of their research through Gene Ontology (GO) analysis with interactive network visualisations and links to related web sites.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;183&quot; data-target=&quot;.183collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Edgar Santos-Fernandez&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;ActisoftR: a toolbox for processing and visualizing scored actigraphy data.&lt;/td&gt;  &lt;td&gt;visualisation, applications, space/time&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 183collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Actigraphy is a cost-effective and convenient tool for activity-based monitoring. It allows studying sleep/wake patterns and identifying disorders in sleep research. ActisoftR was designed for parsing actigraphy outputs and to summarise scored data across user-defined intervals. It consists of several functions for importing, generating reports and statistics, and for data visualization.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;186&quot; data-target=&quot;.186collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Hannah Coughlan&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Integration and visualisation of high throughput genomics data with R&lt;/td&gt;  &lt;td&gt;visualisation, bioinformatics&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 186collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;As sequencing of DNA becomes a more affordable option for studying genomics research questions, more data types are becoming available. One interesting and complex data type, chromosome confirmation capture, aims to interrogate the 3D structure of chromosomes. The same 2 metres of DNA is compacted into the nucleus of every human cell regardless of the cell function, and the genes that determine the cell function are controlled by strict regulation. Chromosome structure can be a mechanism of gene regulation; more specifically, DNA loops form to spatially associate genes with regulators that are not adjacent in the linear genome.However, the techniques to study chromosome structure (Hi-C) are often limited by spatial resolution and can be difficult to interpret. Other genomics techniques that study gene expression (RNA-seq) and regulation (ChIP-seq) cannot discover far away regulators. Here we will show how different types of genomics data can be integrated to investigate long distance gene regulation. Using R and Bioconductor packages (edgeR, Sushi, GenomicRanges and limma) we can integrate data into a common framework that can be visualised to allow for biological interpretation.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;188&quot; data-target=&quot;.188collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Vipavee Trivittayasil&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;MovingBubbles : Animated d3 bubble chart&lt;/td&gt;  &lt;td&gt;visualisation&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 188collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Webpages: https://github.com/chengvt/MovingBubblesA line graph is usually used to portray time-series data. However when there are many time-series, the graph can become cluttered and thus difficult to read.  In order to portray the time-series data with many samples in a more intuitive way, a package for plotting an animated bubble chart was developed. A bubble chart here refers to the chart which represents one quantity but arranged in a way that the bubbles are packed close together to use the space efficiently. The quantity each bubble represents is proportional to the bubble area. There is already a package to plot a static bubble chart in R (Joe Chang et al., n.d.). The MovingBubbles package provides a method to add second and third information dimensions to the chart by means of animation and color. The animation portrays changes in data with time and also helps make the plot more engaging to the viewers. The plotting and transitions between frames are handled by d3 library (Bostock et al., 2011). The package uses htmlwidgets framework (Vaidyanathan et al., 2017) to bridge Javascript and R.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;189&quot; data-target=&quot;.189collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Nicholas Spyrison&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;spinifex: visulizing local structure of higher dimensions&lt;/td&gt;  &lt;td&gt;visualisation, community/education&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 189collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Visualizing in higher (greater than p=3 numeric dimensions) can be messy and unintuitive. Here we explain the methodology and explore the functionality of tourr. We offer a vignette for use and contrast with other higher dimensional visualization methods. The R package, tourr (2011, Wickham, H., D. Cook), gives us the means to animate the projection as we rotate though p-dimensions. This is achieved by varying the contributions from each dimension, via random walk, predefined path, or optimizing an index.Wickham, H., D. Cook, and H. Hofmann (2015). Visualising statistical models: Removing the blindfold (withdiscussion). Statistical Analysis and Data Mining 8(4), 203–225.Wickham, H., D. Cook, H. Hofmann, and A. Buja (2011). tourr: An r package for exploring multivariate data withprojections. Journal of Statistical Software 40(2), http://www.jstatsoft.org/v40.Asimov D (1985). “The Grand Tour: A Tool for Viewing Multidimensional Data.” SIAMJournal of Scientific and Statistical Computing, 6(1), 128–143.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;195&quot; data-target=&quot;.195collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Motoyuki Oki&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Time Series Digger : Automatic time series analysis for data science in R&lt;/td&gt;  &lt;td&gt;visualisation, data mining, space/time&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 195collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Exploratory Data Analysis (EDA) is an essential process for understanding time series and conducting useful feature extraction. We introduce &quot;Time Series Digger&quot;, which provides automatic and programmable EDA in R to accelerate time series analysis for data scientists.Time Series Digger is now deployed on data science platform in NTT Communications which is one of the largest Internet service providers in Japan. We show the effectiveness with real use cases.Time Series Digger consists of three parts.First, it provides automatic and comprehensive time series visualization on various time interval to understand the time series.Second, it provides basic and programmable feature extraction from uni- or multi-variate time series.Third, it applies the features to multiple time series anomaly detection methods.A number of various packages to treat with time series including forecasting and anomaly detection methods exist in R packages.To the best of our knowledge, no packages have focused on efficient and comprehensive analysis process, especially for multiple time series.Our package and contributions should effectively work for R users that face similar problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;29&quot; data-target=&quot;.29collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Volha Tryputsen&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Antibody characterization with next generation sequencing using Group My Abs shiny app&lt;/td&gt;  &lt;td&gt;visualisation, algorithms, models, data mining, applications, web app, reproducibility, bioinformatics&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 29collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Next-generation sequencing (NGS), phage display technology and high throughput capacities enables biologists in drug discovery to characterize antibodies (Abs) based on their HCDR3 sequences and further group them into families before moving to hit-to-lead stage of drug discovery and development. This enables diversification of Ab portfolio and insures back up options if Ab candidate fails. However, there was no method or software available in-house to support Ab discovery with capacities to apply biophysical rules to classify the sequences. Shiny app &quot;Group My Abs&quot; was developed to apply biophysical properties for Ab characterization to the NGS data. Several Multiple Sequence Alignment algorithms implemented in the app enable sequence comparability. A method was developed to evaluate differences between comparable sequences and subsequently classify sequences into families. The app provides custom-made and interactive data visualization, enables refined Ab classification in a mathematical manner, considerably increases efficiency and insures reproducibility. This all decreases bias and enables informative decision making during the hit-to-lead stage in biologics drug discovery.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;143&quot; data-target=&quot;.143collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Gabriel Domingo&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Use of R in Antitrust: The case of the Philippine Competition Commission&lt;/td&gt;  &lt;td&gt;visualisation, models, applications, reproducibility, community/education, Antitrust&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 143collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The use of quantitative analysis of economic data in antitrust is well established. As a new competition agency, the challenges of adopting these analyses at the Philippine Competition Commission is daunting. From competitive enforcement to merger control, the R language empowers our teams of economists in their work.We use R's flexibility and power to clean and model price and demand data when investigating anti-competitive agreements or mergers, and abuses of dominant position. R allows our analysts to rule out various theories of harm in the market, while re-focusing our efforts on specific areas of concern. We use several packages for data modeling, but dplyr and antitrust is particularly useful.When defining the geographic market in merger control, we use R's mapping and plotting packages ggmap, ggplot, leaflet and osrm. These tools determine the scope of a market by pinpointing supplier and consumer locations, illustrating routes, and computing distances and travel times.Finally, we will discuss our efforts to expose more of our economists to R via hands-on training sessions in small teams, and our considerations in using of Rmarkdown to standardize our reports.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;191&quot; data-target=&quot;.191collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Adam Gruer&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Using R and Process Control Charts to Help Hospital Management See The Woods For The Trees&lt;/td&gt;  &lt;td&gt;visualisation&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 191collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The poster describes a project undertaken with the Head of Surgery to introduce Process Control tools and methods to a broader population of hospital managers and executives. It was observed that existing reporting of hospital KPIs was encouraging management and other staff to waste time, energy and analytic resources on variances that were not outside the range or random variance. This is an inefficient use of limited resources. The project involved developing RMarkdown reports and flexdashboards to visualise the variance in the processes being monitored.  CRAN packages such as qichart2 and the tidyverse were selected as useful tools for completing the project as well as consulting literature on Process Control and Lean methodology and contacting other R users in health systems such as the NHS in the UK. Also, important topics such as user interface (UI), user experience (UX) and communication, promotion and education programmes needed to be considered and the poster will highlight how other departments in the hospital with experience in these areas were consulted. This poster discusses the technical and cultural challenges faced and the solutions that were developed.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;194&quot; data-target=&quot;.194collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Sharon Lee&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Shiny EMMIXskew for symmetric and asymmetric mixture modelling&lt;/td&gt;  &lt;td&gt;visualisation, algorithms, models, data mining, applications, web app, multivariate&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 194collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;EMMIXskew allows users to easily fit univariate and multivariate mixture models and perform inference. Designed with a focus on analyzing data that exhibit non-normal features such as asymmetry and heavy-tails, EMMIXskew offers the options of fitting mixtures of skew normal and skew t-distributions in addition to traditional normal and t-mixture models. These models have received increasing attention in recent years due to their powerfulness and flexibility, as witnessed by many applications in fields ranging from biomedicine, imaging, social sciences, to finance.   In this talk, we introduce the EMMIXskew package and its accompanying Shiny app. Its main functionalities will be demonstrated with real-life applications. We will also cover various useful tools included in the package, such as density calculation, mode calculation, random sample generation, error rate calculation, and contour visualization. With the Shiny interface, analysis using these models will become much more accessible for all practitioners and R users.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;108&quot; data-target=&quot;.108collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Alan Pearse&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;SSNDesign -- An R Package for Optimal Designs on Spatial Stream Networks&lt;/td&gt;  &lt;td&gt;models, experimental design; optimal design; spatial stream networks&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 108collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Optimal experimental designs maximise the information gained from limited samples. Optimal designs are paramount when precise predictions or parameter estimates are required but data collection is resource intensive. R packages exist to find optimal designs for a few settings; e.g. AlgDesign and OPDOE. However, to our knowledge, there are no R packages for optimal design problems for stream and river networks. Stream networks provide a unique design challenge due to their branching structure and flow accumulation as water moves downstream. Given these statistical challenges and the importance of healthy freshwater ecosystems, computational tools for designing effective monitoring programs on streams with minimal cost for maximum impact are sorely needed. Here, we present SSNdesign; an R package for finding optimal designs on stream networks. This package relies on the S4 SpatialStreamNetwork object and models implemented in the package SSN. It has functionality for finding optimal designs for estimating model parameters and making predictions on stream networks. Users can also define utility functions for their own design problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;123&quot; data-target=&quot;.123collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Tatiana Marci&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Using Factor Mixture Analysis in Developmental Psychology: An Application to Research on Parent-child Attachment&lt;/td&gt;  &lt;td&gt;applications, Factor Mixture Analysis, heterogeneous populations&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 123collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Factor Mixture Analysis (FMA) is a useful tool to explore data from potentially heterogeneous populations using a crossbred of both categorical and continuous latent variables. Briefly, this approach allows to explore the underlying factorial structure of a theoretical construct, while simultaneously detecting unobserved subgroups in the study population. Thus, FMA becomes particularly useful to investigate psychological phenomena assumed to be categorical and continuous at the same time, and when the source of heterogeneity in the considered population may be not directly observed. Despite these advantages, its application within the psychological sciences remains limited. The current study aims to illustrate the utility of FMA within the context of attachment research in developmental psychology. By presenting a real data example concerning the latent structure of attachment in middle childhood, this work provides a practical example of FMA application using the FactMixtAnalyses package (Viroli, 2011). Furthermore, we will describe ad hoc R functions to assist in the interpretation of results. Benefits and drawbacks of applying FMA to this research area will be discussed.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;11&quot; data-target=&quot;.11collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Gi-Seop Lee&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Evaluations of the machine learning models in the coastal habitat classification&lt;/td&gt;  &lt;td&gt;applications, multivariate, performance&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 11collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;A ‘short-neck clam’ (Ruditapes philippinarum) is one of the most important commercial shellfish. The amount of the shellfish production has been severely reduced due to the unexpected invasion of the ‘Japanese mud shrimp’ (Upogebia major) in some Korean tidal flats. Thus, it is highly required to know the habitat suitability for both organisms. In this study, the diverse simulations of the habitat classification were carried out using the available habitat data of the U. major and R. philippinarum. Supervised learning methods such as decision tree, k-Nearest Neighbor (kNN), Support Vector Machine (SVM) and Artificial Neural Network (ANN) were used with the three optimal clusters defined by R package ‘NbClust’. The decision trees were applied ‘bagging’ and ‘adaboost’ algorithms. Based on the simulation results, the prediction accuracies of each model in case of using the test data are estimated to be about 55-65%. This is considered to be due to outlier effects, and the overfitting problem due to the relatively small number of samples. In many biological data, these are still challenging problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;80&quot; data-target=&quot;.80collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Koji Makiyama&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Magic Functions to Obtain Results from 'for' Loops in R&lt;/td&gt;  &lt;td&gt;interfaces&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 80collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The function 'for' is one of the most popular functions in R. As you know, it is used to create loops. We think there is an inconvenience of 'for' loops in R. It is that the results you get will be gone away. So we have created a package to store the results automatically. To do it, you only need to cast one line spell 'magic_for.' For instance, to calculate squared values for 1 to 3 using 'for' loop and 'print' function is very easy. However, it becomes too much hassle to change such codes to store displayed results. You must prepare some containers with correct length for storing results and change 'print' function to assignment statements. Moreover, in such or more troublesome situations like where you have to store many variables, codes will grow more complex. The 'magicfor' package makes to resolve the problem with keep readability. You just add one line 'magic_for()' before 'for' loops. Once you call 'magic_for,' you can just execute 'for' as usual, the results will be stored in memory automatically. You can obtain the results using 'magic_result.' We introduce how to use the magic.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;155&quot; data-target=&quot;.155collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Yuya Matsumura&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Easy Writing of Bayesian Optimizaion for Macine Learning&lt;/td&gt;  &lt;td&gt;models, performance, big data&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 155collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In many machine learning algorithms, tuning  hyperparameters is one of the most important point. Bayesian optimization (Shahriari et al., 2015) is a method for tuning hyperparameters faster and more efficient than grid search that searches all grids in parameter space. In R, combination of rBayesianOptimization package and some machine learning packages such as e1071 or ranger enable Bayesian optimization for hyperparameter tuning. However, it was troublesome to write codes for Bayesian optimization using those packages because we must make a complicated function to maximize, then write as code to execute Bayesian optimization. This is very confusing, hard to try and try and error. MlBayesOpt package (Matsumura, 2017, https://cran.r-project.org/web/packages/MlBayesOpt/index.html) is a very convenient to write this work. This package requires to execute Bayesian optimization only a dataframe, column name of label to classify (or regress), and column names of feature vectors. For example, there are 32 lines of a source using combination of packages, but 5 lines of that using MlBayesOpt package.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;161&quot; data-target=&quot;.161collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Jeremy Forbes&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Using Australian census data to describe electorates' socio-economic profiles at the time of a federal election.&lt;/td&gt;  &lt;td&gt;models, reproducibility, community/education&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 161collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In Australia, the House of Representatives is divided into 150 seats, each representing an electoral division, and each divisions' boundaries are revised periodically. Federal elections generally occur every three years, but electorate boundaries can change in between elections.The Australian Bureau of Statistics conducts a Census of Population and Housing every five years, and updates its record of electorate boundaries in July each year, in accordance with the official electoral commission's boundaries.This research looks at matching and estimating the socio-economic profile of each electorate at the time of a federal election.To accurately estimate profiles, each election is initially paired with the Census data taken closet to the election date. Many elections do not occur in the same year as a census, and are matched with data from nearby years. Differences between these dates are adjusted for using spatial analysis and time-series forecasts.This work is an update for the eechidna package, which contains Australian census and election data, and tools for visualisation and analysis.PS. An update can be provided closer to date. Research has only recently commenced.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;163&quot; data-target=&quot;.163collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Jessica Bagnall&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Analysing the voting patterns of the Senate of the 45th Australian Parliament via fully-visible Boltzmann machines&lt;/td&gt;  &lt;td&gt;algorithms, models, applications, networks&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 163collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The 45th Australian Senate—following the 2016 federal election—contains the largest crossbench since the expansion of the Senate in 1950. Of the 20 Senators who make up the crossbench, 7 minor parties were elected.We analyse the party-level voting patterns of the parties of the Senate of the 45th Australian parliament by modelling the crossbench via a fully-visible Boltzmann machine, a probabilistic graphical network that arises from the neural networks literature, in order to determine the various influences that each party has on each other, and to evaluate the relative pro- (or anti-) government stances of the aforementioned parties.We describe the required estimations and computations that are performed via our R package BoltzMM—available at github.com/andrewthomasjones/BoltzMM. The package implements the MM algorithm for maximum pseudolikelihood estimation of FVBM models of Nguyen and Wood (Neural Computation, 2016), and uses the asymptotic normality results of Nguyen and Wood (IEEE T Neural Networks and Learning Systems, 2016) for inferential computations.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;168&quot; data-target=&quot;.168collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Florian Schwendinger&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Readability Prediction in R&lt;/td&gt;  &lt;td&gt;models, applications, text analysis/NLP&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 168collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Readability prediction is commonly used to assess the comprehensibility of a given text. Early approaches focus on the development of readability scores (e.g.  Fog-Index, Dale-Chall, Flesch Reading Ease). Most of these readability scores are based on the number of words, number of sentences, number of syllables and number of words which are not present in a predefined list.Current research in the field of linguistics suggests that these scores are often misleading and models which combine Natural Language Processing (NLP) and statistical learning should be used instead.This research presents how a state-of-the-art readability prediction can be implemented in R by utilizing the tools available from the StanfordCoreNLP package. The StanfordCoreNLP package and its companions can be installed from https://datacube.wu.ac.at/.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;178&quot; data-target=&quot;.178collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Aswi Aswi&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Comparison of different Bayesian spatio-temporal models using R packages&lt;/td&gt;  &lt;td&gt;models, applications, space/time, CARBayesST&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 178collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;There is a growing number of packages in R for modelling spatio-temporal data. In this presentation, we will review and compare a number of spatio-temporal Bayesian models using R. We will focus on two R packages, namely R-INLA (Integrated Nested Laplace Approximation) and CARBayesST and describe the different spatio-temporal models available. We examine six and five Bayesian spatio temporal models using CARBayes and R-INLA, respectively. We will illustrate the application of these models and packages through a case study on dengue cases, in Makassar, Indonesia. Model performance will be compared using goodness of fit such as Deviance Information Criteria (DIC). The computational speed and ease of using these packages makes them a very attractive option for Bayesian spatio-temporal modelling.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;211&quot; data-target=&quot;.211collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Janek Thomas&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Automatic gradient boosting&lt;/td&gt;  &lt;td&gt;algorithms, models, data mining, Automatic Machine Learning&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 211collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Well-qualified data scientist are not a dime a dozen. Instead, employees being not very familiar with data analysis are often called to do the job. Automatic machine learning can help those persons to perform predictive modeling with high performing machine learning tools without having much experience. This is achieved by making those applications parameter-free, i.e. only the data is required as input. Projects like Auto-WEKA or auto-sklearn aim to solve the Combined Algorithm Selection and Hyperparameter optimization (CASH) problem resulting in a huge optimization space. However, for most real world applications, only few different learning algorithms are required to deliver superior performances. autoxgboost simplifies this idea one step further and the CASH problem to taking Gradient Boosting as a single learning algorithm in combination with intelligent model based hyperparameter tuning. It is based on the R-Packages mlr, mlrMBO and XGBoost. It also supports categorical variables due to special inbuilt factor feature encoding. Even though autoxgboost only uses one learner instead of a whole library, it provides comparable or even better performances.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;212&quot; data-target=&quot;.212collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Awdhesh Yadav&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Household and Community factor on under-five mortality in India: An application of multilevel cox proportional hazard model&lt;/td&gt;  &lt;td&gt;multivariate, big data&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 212collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The objective of this paper is to determine the important of community, household and individual level effect on under-five mortality in India. Using data from the latest round of Demographic Health Survey (DHS)-2005-06, multilevel cox proportional hazard analysis was performed on a nationally representative sample. The results indicate that pattern of under-five mortality were clustered within mothers and communities. The community level variables like region, place of residence, community poverty level, community education level, ethnic fractionalization index were significantly determine under-five mortality in India. The risk of under-five deaths was significantly higher for children residing in North, East and West regions compared to South region. In addition, the proportion of women in community completing secondary school were significantly more likely to increase the child survival. The household level variables like religion, caste and wealth index were significantly determining under-five mortality. The results suggest to address the contextual level factors to address under-five mortality in India&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;21&quot; data-target=&quot;.21collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Jean-Michel Perraud&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;A suite of R packages for hydrological ensemble forecasting using Rcpp&lt;/td&gt;  &lt;td&gt;space/time, performance, big data, Hydrology, forecast&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 21collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Ensemble prediction techniques have been shown to produce more accurate predictions as well as formally quantify prediction uncertainty in a range of scientific applications. We present a suite of libraries for hydrological ensemble forecasts designed for use both in research and operations. The features of the C++ libraries are available from several high-level interactive languages including R. The suite currently comprises three main R packages for rainfall forecast post processing (RPP), semi-distributed ensemble hydrological modelling (SWIFT2) and multi-dimensional ensemble time series (uchronia). The packages are designed to offer concise commands for handling ensemble time series, input/output, model parameterisation and simulation execution. The native libraries purposely have a C API for maximising interoperability and foster a consistent use experience across high-level languages. Rcpp is used for surfacing the features in the R packages. Bespoke code for marshaling data, object lifetime management and generating glue code for Rcpp is already open source and suitable for reuse in similar technical contexts.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;67&quot; data-target=&quot;.67collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Susanna Cramb&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Bayesian disease mapping in R&lt;/td&gt;  &lt;td&gt;models, space/time&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 67collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Bayesian methods prominently feature in disease mapping, and R has multiple packages designed to enable efficient computation of assorted Bayesian spatial models.Here we examine seven Bayesian models suitable for disease mapping and implement them using the R packages of R-INLA, CARBayes, R2WinBUGS and R2jags. Models considered included common approaches such as the BYM which smooth estimates over all adjacent areas, through to more recently introduced models that allowed for discontinuities between adjacent areas, as well as spline models. Simulated incidence data designed to represent a rare cancer (liver) and more common cancer (lung) were examined across 2153 areas in Australia. Model performance was compared on goodness of fit measures (WAIC, Moran’s I on residuals), computational time and convergence (Geweke). The packages themselves are also compared in terms of computational time and model flexibility. It is useful to consider several different models to understand the robustness of results when disease mapping. R has the capacity to enable a wide range of models to be considered, with the additional advantages of high quality visualisation of results.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;196&quot; data-target=&quot;.196collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Rosemary Putler&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;Analysis of EHR Data and Circulating Inflammatory Mediators: Association with Severe Clostridium difficile Infection&lt;/td&gt;  &lt;td&gt;models, applications, bioinformatics&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 196collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Clostridium difficile infection (CDI) is a major healthcare-associated infection, and severe CDI often leads to subsequent recurrence or death. We hypothesized that circulating inflammatory mediators would associate with severity in a prospective cohort of inpatients diagnosed with CDI. An inflammatory mediator panel was performed on collected sera and merged with electronic health record (EHR) data. With these data we show that circulating biomarkers associate not only with severity of the CDI episode, but also with subsequent mortality. Because of the large number of potentially correlated predictors, our data presented a challenge when we set out to identify features to incorporate into an accurate predictive model. We explore the steps performed in this analysis, discussing methodology and decision points, including the management and analysis of EHR data, utilization of dimensional reduction techniques, and use of existing packages such as vegan, glmnet, and pROC. Through this analysis, we demonstrate how a diverse array of R packages and statistical methodologies, which function in a wide array of use cases, can also be used to answer a complex disease-related question.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;200&quot; data-target=&quot;.200collapsed&quot;&gt;  &lt;td&gt;14:40&lt;/td&gt;  &lt;td&gt;Poster lightning&lt;/td&gt;  &lt;td&gt;Stuart Davie&lt;/td&gt;  &lt;td&gt;AUD&lt;/td&gt;  &lt;td&gt;A data driven approach to generating and scoring B2B leads&lt;/td&gt;  &lt;td&gt;visualisation, models, applications&lt;/td&gt;  &lt;td&gt;Simon Jackson&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 200collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In many industries, companies rely on a sales team to source and qualify leads. Unfortunately, this limits a company's potential leads to those that can be manually processed, while lead qualification is limited by the quality of ad hoc scoring systems. To find leads faster, companies might engage in cold-calling, or blanket email campaigns, both of which are known for their low conversion rates. Here, a data-driven B2B lead generation and qualification solution for the UK market is presented, based on open source data and XGBoost. Our models take into account both general and company specific features, and allow an approximation of the size of market opportunity. Lead reports containing pertinent conversion information are automatically generated using xgboostExplainer and R Markdown.  Considerations on feature engineering, and difficulties associated with overfitting, are also discussed. **As there are several components to this presentation, a lightning talk would be preferred over a poster**&lt;/td&gt;&lt;/tr&gt;
&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;36&quot; data-target=&quot;.36collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Shinichi Takayanagi&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;How LINE Corp Use R to Compete in a Data-Driven World&lt;/td&gt;  &lt;td&gt;applications, reproducibility, Practical use of R language in business&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 36collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;LINE is one of the most popular messaging applications in Asia developed by LINE corp.We always collect and utilize the log of user behavior to import our services.In our data science team, R plays an important role in all stages of the data-analysis and science.It ranges from exploratory data analysis and modeling to sharing the results using Shiny with other colleagues.In our poster presentation, we explain how we use R to solve our business problems and share some practical insights for others who want to use R in their business.At first, as our data science team grows, different people develop their own R script to solve similar tasks like data base connection.We started working on our internal R package called &quot;liner&quot; shared through Github Enterprise to solve this problems.Now, multiple members often collaborate in order to improve and fix bugs using this package.We also introduce an overview of our R-related data analysis platform along with some useful OSS tools:- yanagishima: It helps people run query easily developed at LINE corp- Drone: Docker-based continuous integration tool. We use this for automated unit test and deploy &quot;liner&quot; package to users&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;63&quot; data-target=&quot;.63collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Mahmoud Ahmed&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;pcr: an R package for quality assessment, analysis and testing of qPCR data&lt;/td&gt;  &lt;td&gt;bioinformatics, data-access&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 63collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Real-time quantitative PCR (qPCR) is a broadly used technique in the biomedical research. We developed an R package to implement methods for quality assessment, analysis and testing qPCR data for statistical significance. The Double Delta CT and standard curve models were implemented to quantify the relative expression of target genes from CT values in standard qPCR. In addition, calculation of amplification efficiency and curves from serial dilution qPCR experiments are used to assess the data quality. Finally, two-group testing and linear models are used to test for statistical significance among conditions. Using two datasets from qPCR experiments, we applied different quality assessment, analysis and statistical testing in the pcr package and compared the results to the original published articles. The pcr package provides an intuitive and unified interface for its main functions to allow biologist to perform all necessary steps of qPCR analysis and produce graphs in a uniform way.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;69&quot; data-target=&quot;.69collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Miles McBain&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;An R pipeline for creating and hosting collaborative Web VR environments.&lt;/td&gt;  &lt;td&gt;visualisation, space/time&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 69collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;We are continuing the rich history of R as a generative tool for data driven documents by developing a capability to generate Virtual Reality (VR) scenes. A VR scene object is provided that can create, configure, and serve multi-user Web VR scenes. Development so far includes tools that harness the R spatial ecosystem to create textured 3D meshes from contour and raster data. Several other VR primitives are provided, including 360 photo 'portals'.The immediate application for this work has been the creation of environments for the calibration of geo-spatial statistical models by expert elicitation. However, we see the ability to generate and host multi-user scenes, complete with facility for activity logging, as a compelling new platform for collecting and visualising experimental data.Given this work is evolving rapidly, a poster would be the perfect tool to engage attendees in a variety of conversations about the API, the technology stack, and applications. If given a poster slot at useR!2018, we would like to bring some VR hardware to the session, to give attendees a chance to experience some of the virtual environments our tools have generated. &lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;6&quot; data-target=&quot;.6collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Kim Ki-Yeol&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Squamous cell carcinoma analysis with R&lt;/td&gt;  &lt;td&gt;bioinformatics&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 6collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Squamous cell carcinoma (SCC) is the most common histological type of head neck cancer and cervical cancer. Carcinogenesis in these two types of cancers demonstrate similar multistep progression. The purpose of the study is to identify the significant consensus gene modules of these two cancers. We used a publicly available expression dataset for the study. The dataset included head neck cancer (42 cancer samples and 14 normal samples) and cervical cancer (20 cancer samples and 8 normal samples). We used only human papilloma virus 16 positive samples for excluding the bias according to the different types of HPV. We identified consensus modules of two types of cancers and explored the biological functions of each modules by annotation tool. We identified 8 consensus gene modules of head neck cancer and cervical cancer. Each module was well preserved between the two types of cancer. The modules included significant biological functions, including ATP binding and extracellular exosome. Consensus gene module identification is expected to contribute to more personalized management of multiple cancer types.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;95&quot; data-target=&quot;.95collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Erika Siregar&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;AnalevR: An Interactive R-Based Analysis Environment for Utilizing BPS-Statistics Indonesia Data&lt;/td&gt;  &lt;td&gt;visualisation, web app, community/education, networks, big data, analysis, collaboration, cloud computing, analysis as a service, bps, statistics Indonesia, analysis environment, remote data analysis&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 95collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;As Indonesia’s national statistical agency, BPS-Statistics Indonesia produces a massive amount of strategic data every year. However, these data are still underutilized by other parties (governments, researchers, etc.) due to technical limitations and raw data exclusivity and locality. Actually, numerous people outside BPS are capable of conducting analysis but unable to access the data. To increase the data usefulness, we introduce AnalevR, an online R-based analysis environment that allows anyone to perform analyses and create visualization without having to own the original raw data. It uses a notebook-like interface where users can type commands and the output appears below it. BPS provides the data and analysis service (including the R modules) which are held in cloud storage and can be explored via the helper function. Users remotely execute R commands and perform analysis inside the workspaces. A user can create up to 10 workspaces, each representing different sessions. Each saved session preserves the user-defined variables and functions for future use. This breakthrough will raise users’ involvement in employing BPS’ data and increase statistical quality in Indonesia.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;111&quot; data-target=&quot;.111collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Yan Holtz&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;From Data to Viz&lt;/td&gt;  &lt;td&gt;visualisation, community/education&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 111collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Selecting the right graphic type is a common task for a data scientist. On a daily basis, an R user deals with a data frame and must decide what visualization is the most appropriate to represent it.The task is not easy. The data scientist must:-	Know the broad spectrum of visualization types-	Figure out which dataviz is doable given the dataset-	Try several (or all) of them-	Find the code to create the charts-	Avoid the common caveats associated with the selected optionData-to-viz.com is a new website that comes to meet these needs. It displays an interactive decision tree. The user describe their dataset, what leads them to a set of appropriate graphic types. A description is provided for each, explaining its pros and cons. Links to the R and the Python graph galleries are provided, which allows to get the corresponding code in seconds.The complete decision tree is also available in a static version through a poster. The project has not been released yet due to its potential announcement at the useR conference.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;148&quot; data-target=&quot;.148collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Goknur Giner&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Pathway-VisualiseR&lt;/td&gt;  &lt;td&gt;visualisation, web app, community/education, bioinformatics, networks, Bioconductor&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 148collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Statistical modelling of any genomic research produces sets of genes and individual biomarkers which require investigation. Further  exploration of those set of biomarkers is a prominent step towards  discovering the source of a biological problem. Furthermore,  understanding the collective behaviour of genes has been shown to provide valuable insights into the triggers of many human diseases. We  have developed an RShiny application that provides an interface for  researchers, enabling them to discover the interaction between their  genes and biological pathways. This application allow users to inquire  into details of their research through Gene Ontology (GO) analysis with interactive network visualisations and links to related web sites.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;183&quot; data-target=&quot;.183collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Edgar Santos-Fernandez&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;ActisoftR: a toolbox for processing and visualizing scored actigraphy data.&lt;/td&gt;  &lt;td&gt;visualisation, applications, space/time&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 183collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Actigraphy is a cost-effective and convenient tool for activity-based monitoring. It allows studying sleep/wake patterns and identifying disorders in sleep research. ActisoftR was designed for parsing actigraphy outputs and to summarise scored data across user-defined intervals. It consists of several functions for importing, generating reports and statistics, and for data visualization.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;186&quot; data-target=&quot;.186collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Hannah Coughlan&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Integration and visualisation of high throughput genomics data with R&lt;/td&gt;  &lt;td&gt;visualisation, bioinformatics&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 186collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;As sequencing of DNA becomes a more affordable option for studying genomics research questions, more data types are becoming available. One interesting and complex data type, chromosome confirmation capture, aims to interrogate the 3D structure of chromosomes. The same 2 metres of DNA is compacted into the nucleus of every human cell regardless of the cell function, and the genes that determine the cell function are controlled by strict regulation. Chromosome structure can be a mechanism of gene regulation; more specifically, DNA loops form to spatially associate genes with regulators that are not adjacent in the linear genome.However, the techniques to study chromosome structure (Hi-C) are often limited by spatial resolution and can be difficult to interpret. Other genomics techniques that study gene expression (RNA-seq) and regulation (ChIP-seq) cannot discover far away regulators. Here we will show how different types of genomics data can be integrated to investigate long distance gene regulation. Using R and Bioconductor packages (edgeR, Sushi, GenomicRanges and limma) we can integrate data into a common framework that can be visualised to allow for biological interpretation.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;188&quot; data-target=&quot;.188collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Vipavee Trivittayasil&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;MovingBubbles : Animated d3 bubble chart&lt;/td&gt;  &lt;td&gt;visualisation&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 188collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Webpages: https://github.com/chengvt/MovingBubblesA line graph is usually used to portray time-series data. However when there are many time-series, the graph can become cluttered and thus difficult to read.  In order to portray the time-series data with many samples in a more intuitive way, a package for plotting an animated bubble chart was developed. A bubble chart here refers to the chart which represents one quantity but arranged in a way that the bubbles are packed close together to use the space efficiently. The quantity each bubble represents is proportional to the bubble area. There is already a package to plot a static bubble chart in R (Joe Chang et al., n.d.). The MovingBubbles package provides a method to add second and third information dimensions to the chart by means of animation and color. The animation portrays changes in data with time and also helps make the plot more engaging to the viewers. The plotting and transitions between frames are handled by d3 library (Bostock et al., 2011). The package uses htmlwidgets framework (Vaidyanathan et al., 2017) to bridge Javascript and R.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;189&quot; data-target=&quot;.189collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Nicholas Spyrison&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;spinifex: visulizing local structure of higher dimensions&lt;/td&gt;  &lt;td&gt;visualisation, community/education&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 189collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Visualizing in higher (greater than p=3 numeric dimensions) can be messy and unintuitive. Here we explain the methodology and explore the functionality of tourr. We offer a vignette for use and contrast with other higher dimensional visualization methods. The R package, tourr (2011, Wickham, H., D. Cook), gives us the means to animate the projection as we rotate though p-dimensions. This is achieved by varying the contributions from each dimension, via random walk, predefined path, or optimizing an index.Wickham, H., D. Cook, and H. Hofmann (2015). Visualising statistical models: Removing the blindfold (withdiscussion). Statistical Analysis and Data Mining 8(4), 203–225.Wickham, H., D. Cook, H. Hofmann, and A. Buja (2011). tourr: An r package for exploring multivariate data withprojections. Journal of Statistical Software 40(2), http://www.jstatsoft.org/v40.Asimov D (1985). “The Grand Tour: A Tool for Viewing Multidimensional Data.” SIAMJournal of Scientific and Statistical Computing, 6(1), 128–143.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;195&quot; data-target=&quot;.195collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Motoyuki Oki&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Time Series Digger : Automatic time series analysis for data science in R&lt;/td&gt;  &lt;td&gt;visualisation, data mining, space/time&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 195collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Exploratory Data Analysis (EDA) is an essential process for understanding time series and conducting useful feature extraction. We introduce &quot;Time Series Digger&quot;, which provides automatic and programmable EDA in R to accelerate time series analysis for data scientists.Time Series Digger is now deployed on data science platform in NTT Communications which is one of the largest Internet service providers in Japan. We show the effectiveness with real use cases.Time Series Digger consists of three parts.First, it provides automatic and comprehensive time series visualization on various time interval to understand the time series.Second, it provides basic and programmable feature extraction from uni- or multi-variate time series.Third, it applies the features to multiple time series anomaly detection methods.A number of various packages to treat with time series including forecasting and anomaly detection methods exist in R packages.To the best of our knowledge, no packages have focused on efficient and comprehensive analysis process, especially for multiple time series.Our package and contributions should effectively work for R users that face similar problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;29&quot; data-target=&quot;.29collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Volha Tryputsen&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Antibody characterization with next generation sequencing using Group My Abs shiny app&lt;/td&gt;  &lt;td&gt;visualisation, algorithms, models, data mining, applications, web app, reproducibility, bioinformatics&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 29collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Next-generation sequencing (NGS), phage display technology and high throughput capacities enables biologists in drug discovery to characterize antibodies (Abs) based on their HCDR3 sequences and further group them into families before moving to hit-to-lead stage of drug discovery and development. This enables diversification of Ab portfolio and insures back up options if Ab candidate fails. However, there was no method or software available in-house to support Ab discovery with capacities to apply biophysical rules to classify the sequences. Shiny app &quot;Group My Abs&quot; was developed to apply biophysical properties for Ab characterization to the NGS data. Several Multiple Sequence Alignment algorithms implemented in the app enable sequence comparability. A method was developed to evaluate differences between comparable sequences and subsequently classify sequences into families. The app provides custom-made and interactive data visualization, enables refined Ab classification in a mathematical manner, considerably increases efficiency and insures reproducibility. This all decreases bias and enables informative decision making during the hit-to-lead stage in biologics drug discovery.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;143&quot; data-target=&quot;.143collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Gabriel Domingo&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Use of R in Antitrust: The case of the Philippine Competition Commission&lt;/td&gt;  &lt;td&gt;visualisation, models, applications, reproducibility, community/education, Antitrust&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 143collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The use of quantitative analysis of economic data in antitrust is well established. As a new competition agency, the challenges of adopting these analyses at the Philippine Competition Commission is daunting. From competitive enforcement to merger control, the R language empowers our teams of economists in their work.We use R's flexibility and power to clean and model price and demand data when investigating anti-competitive agreements or mergers, and abuses of dominant position. R allows our analysts to rule out various theories of harm in the market, while re-focusing our efforts on specific areas of concern. We use several packages for data modeling, but dplyr and antitrust is particularly useful.When defining the geographic market in merger control, we use R's mapping and plotting packages ggmap, ggplot, leaflet and osrm. These tools determine the scope of a market by pinpointing supplier and consumer locations, illustrating routes, and computing distances and travel times.Finally, we will discuss our efforts to expose more of our economists to R via hands-on training sessions in small teams, and our considerations in using of Rmarkdown to standardize our reports.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;191&quot; data-target=&quot;.191collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Adam Gruer&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Using R and Process Control Charts to Help Hospital Management See The Woods For The Trees&lt;/td&gt;  &lt;td&gt;visualisation&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 191collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The poster describes a project undertaken with the Head of Surgery to introduce Process Control tools and methods to a broader population of hospital managers and executives. It was observed that existing reporting of hospital KPIs was encouraging management and other staff to waste time, energy and analytic resources on variances that were not outside the range or random variance. This is an inefficient use of limited resources. The project involved developing RMarkdown reports and flexdashboards to visualise the variance in the processes being monitored.  CRAN packages such as qichart2 and the tidyverse were selected as useful tools for completing the project as well as consulting literature on Process Control and Lean methodology and contacting other R users in health systems such as the NHS in the UK. Also, important topics such as user interface (UI), user experience (UX) and communication, promotion and education programmes needed to be considered and the poster will highlight how other departments in the hospital with experience in these areas were consulted. This poster discusses the technical and cultural challenges faced and the solutions that were developed.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;194&quot; data-target=&quot;.194collapsed&quot;&gt;  &lt;td&gt;18:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Sharon Lee&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Shiny EMMIXskew for symmetric and asymmetric mixture modelling&lt;/td&gt;  &lt;td&gt;visualisation, algorithms, models, data mining, applications, web app, multivariate&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 194collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;EMMIXskew allows users to easily fit univariate and multivariate mixture models and perform inference. Designed with a focus on analyzing data that exhibit non-normal features such as asymmetry and heavy-tails, EMMIXskew offers the options of fitting mixtures of skew normal and skew t-distributions in addition to traditional normal and t-mixture models. These models have received increasing attention in recent years due to their powerfulness and flexibility, as witnessed by many applications in fields ranging from biomedicine, imaging, social sciences, to finance.   In this talk, we introduce the EMMIXskew package and its accompanying Shiny app. Its main functionalities will be demonstrated with real-life applications. We will also cover various useful tools included in the package, such as density calculation, mode calculation, random sample generation, error rate calculation, and contour visualization. With the Shiny interface, analysis using these models will become much more accessible for all practitioners and R users.&lt;/td&gt;&lt;/tr&gt;
</code></pre>
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<pre><code>&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;108&quot; data-target=&quot;.108collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Alan Pearse&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;SSNDesign -- An R Package for Optimal Designs on Spatial Stream Networks&lt;/td&gt;  &lt;td&gt;models, experimental design; optimal design; spatial stream networks&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 108collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Optimal experimental designs maximise the information gained from limited samples. Optimal designs are paramount when precise predictions or parameter estimates are required but data collection is resource intensive. R packages exist to find optimal designs for a few settings; e.g. AlgDesign and OPDOE. However, to our knowledge, there are no R packages for optimal design problems for stream and river networks. Stream networks provide a unique design challenge due to their branching structure and flow accumulation as water moves downstream. Given these statistical challenges and the importance of healthy freshwater ecosystems, computational tools for designing effective monitoring programs on streams with minimal cost for maximum impact are sorely needed. Here, we present SSNdesign; an R package for finding optimal designs on stream networks. This package relies on the S4 SpatialStreamNetwork object and models implemented in the package SSN. It has functionality for finding optimal designs for estimating model parameters and making predictions on stream networks. Users can also define utility functions for their own design problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;123&quot; data-target=&quot;.123collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Tatiana Marci&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Using Factor Mixture Analysis in Developmental Psychology: An Application to Research on Parent-child Attachment&lt;/td&gt;  &lt;td&gt;applications, Factor Mixture Analysis, heterogeneous populations&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 123collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Factor Mixture Analysis (FMA) is a useful tool to explore data from potentially heterogeneous populations using a crossbred of both categorical and continuous latent variables. Briefly, this approach allows to explore the underlying factorial structure of a theoretical construct, while simultaneously detecting unobserved subgroups in the study population. Thus, FMA becomes particularly useful to investigate psychological phenomena assumed to be categorical and continuous at the same time, and when the source of heterogeneity in the considered population may be not directly observed. Despite these advantages, its application within the psychological sciences remains limited. The current study aims to illustrate the utility of FMA within the context of attachment research in developmental psychology. By presenting a real data example concerning the latent structure of attachment in middle childhood, this work provides a practical example of FMA application using the FactMixtAnalyses package (Viroli, 2011). Furthermore, we will describe ad hoc R functions to assist in the interpretation of results. Benefits and drawbacks of applying FMA to this research area will be discussed.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;11&quot; data-target=&quot;.11collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Gi-Seop Lee&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Evaluations of the machine learning models in the coastal habitat classification&lt;/td&gt;  &lt;td&gt;applications, multivariate, performance&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 11collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;A ‘short-neck clam’ (Ruditapes philippinarum) is one of the most important commercial shellfish. The amount of the shellfish production has been severely reduced due to the unexpected invasion of the ‘Japanese mud shrimp’ (Upogebia major) in some Korean tidal flats. Thus, it is highly required to know the habitat suitability for both organisms. In this study, the diverse simulations of the habitat classification were carried out using the available habitat data of the U. major and R. philippinarum. Supervised learning methods such as decision tree, k-Nearest Neighbor (kNN), Support Vector Machine (SVM) and Artificial Neural Network (ANN) were used with the three optimal clusters defined by R package ‘NbClust’. The decision trees were applied ‘bagging’ and ‘adaboost’ algorithms. Based on the simulation results, the prediction accuracies of each model in case of using the test data are estimated to be about 55-65%. This is considered to be due to outlier effects, and the overfitting problem due to the relatively small number of samples. In many biological data, these are still challenging problems.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;80&quot; data-target=&quot;.80collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Koji Makiyama&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Magic Functions to Obtain Results from 'for' Loops in R&lt;/td&gt;  &lt;td&gt;interfaces&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 80collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The function 'for' is one of the most popular functions in R. As you know, it is used to create loops. We think there is an inconvenience of 'for' loops in R. It is that the results you get will be gone away. So we have created a package to store the results automatically. To do it, you only need to cast one line spell 'magic_for.' For instance, to calculate squared values for 1 to 3 using 'for' loop and 'print' function is very easy. However, it becomes too much hassle to change such codes to store displayed results. You must prepare some containers with correct length for storing results and change 'print' function to assignment statements. Moreover, in such or more troublesome situations like where you have to store many variables, codes will grow more complex. The 'magicfor' package makes to resolve the problem with keep readability. You just add one line 'magic_for()' before 'for' loops. Once you call 'magic_for,' you can just execute 'for' as usual, the results will be stored in memory automatically. You can obtain the results using 'magic_result.' We introduce how to use the magic.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;155&quot; data-target=&quot;.155collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Yuya Matsumura&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Easy Writing of Bayesian Optimizaion for Macine Learning&lt;/td&gt;  &lt;td&gt;models, performance, big data&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 155collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In many machine learning algorithms, tuning  hyperparameters is one of the most important point. Bayesian optimization (Shahriari et al., 2015) is a method for tuning hyperparameters faster and more efficient than grid search that searches all grids in parameter space. In R, combination of rBayesianOptimization package and some machine learning packages such as e1071 or ranger enable Bayesian optimization for hyperparameter tuning. However, it was troublesome to write codes for Bayesian optimization using those packages because we must make a complicated function to maximize, then write as code to execute Bayesian optimization. This is very confusing, hard to try and try and error. MlBayesOpt package (Matsumura, 2017, https://cran.r-project.org/web/packages/MlBayesOpt/index.html) is a very convenient to write this work. This package requires to execute Bayesian optimization only a dataframe, column name of label to classify (or regress), and column names of feature vectors. For example, there are 32 lines of a source using combination of packages, but 5 lines of that using MlBayesOpt package.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;161&quot; data-target=&quot;.161collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Jeremy Forbes&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Using Australian census data to describe electorates' socio-economic profiles at the time of a federal election.&lt;/td&gt;  &lt;td&gt;models, reproducibility, community/education&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 161collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In Australia, the House of Representatives is divided into 150 seats, each representing an electoral division, and each divisions' boundaries are revised periodically. Federal elections generally occur every three years, but electorate boundaries can change in between elections.The Australian Bureau of Statistics conducts a Census of Population and Housing every five years, and updates its record of electorate boundaries in July each year, in accordance with the official electoral commission's boundaries.This research looks at matching and estimating the socio-economic profile of each electorate at the time of a federal election.To accurately estimate profiles, each election is initially paired with the Census data taken closet to the election date. Many elections do not occur in the same year as a census, and are matched with data from nearby years. Differences between these dates are adjusted for using spatial analysis and time-series forecasts.This work is an update for the eechidna package, which contains Australian census and election data, and tools for visualisation and analysis.PS. An update can be provided closer to date. Research has only recently commenced.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;163&quot; data-target=&quot;.163collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Jessica Bagnall&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Analysing the voting patterns of the Senate of the 45th Australian Parliament via fully-visible Boltzmann machines&lt;/td&gt;  &lt;td&gt;algorithms, models, applications, networks&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 163collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The 45th Australian Senate—following the 2016 federal election—contains the largest crossbench since the expansion of the Senate in 1950. Of the 20 Senators who make up the crossbench, 7 minor parties were elected.We analyse the party-level voting patterns of the parties of the Senate of the 45th Australian parliament by modelling the crossbench via a fully-visible Boltzmann machine, a probabilistic graphical network that arises from the neural networks literature, in order to determine the various influences that each party has on each other, and to evaluate the relative pro- (or anti-) government stances of the aforementioned parties.We describe the required estimations and computations that are performed via our R package BoltzMM—available at github.com/andrewthomasjones/BoltzMM. The package implements the MM algorithm for maximum pseudolikelihood estimation of FVBM models of Nguyen and Wood (Neural Computation, 2016), and uses the asymptotic normality results of Nguyen and Wood (IEEE T Neural Networks and Learning Systems, 2016) for inferential computations.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;168&quot; data-target=&quot;.168collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Florian Schwendinger&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Readability Prediction in R&lt;/td&gt;  &lt;td&gt;models, applications, text analysis/NLP&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 168collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Readability prediction is commonly used to assess the comprehensibility of a given text. Early approaches focus on the development of readability scores (e.g.  Fog-Index, Dale-Chall, Flesch Reading Ease). Most of these readability scores are based on the number of words, number of sentences, number of syllables and number of words which are not present in a predefined list.Current research in the field of linguistics suggests that these scores are often misleading and models which combine Natural Language Processing (NLP) and statistical learning should be used instead.This research presents how a state-of-the-art readability prediction can be implemented in R by utilizing the tools available from the StanfordCoreNLP package. The StanfordCoreNLP package and its companions can be installed from https://datacube.wu.ac.at/.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;178&quot; data-target=&quot;.178collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Aswi Aswi&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Comparison of different Bayesian spatio-temporal models using R packages&lt;/td&gt;  &lt;td&gt;models, applications, space/time, CARBayesST&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 178collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;There is a growing number of packages in R for modelling spatio-temporal data. In this presentation, we will review and compare a number of spatio-temporal Bayesian models using R. We will focus on two R packages, namely R-INLA (Integrated Nested Laplace Approximation) and CARBayesST and describe the different spatio-temporal models available. We examine six and five Bayesian spatio temporal models using CARBayes and R-INLA, respectively. We will illustrate the application of these models and packages through a case study on dengue cases, in Makassar, Indonesia. Model performance will be compared using goodness of fit such as Deviance Information Criteria (DIC). The computational speed and ease of using these packages makes them a very attractive option for Bayesian spatio-temporal modelling.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;211&quot; data-target=&quot;.211collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Janek Thomas&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Automatic gradient boosting&lt;/td&gt;  &lt;td&gt;algorithms, models, data mining, Automatic Machine Learning&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 211collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Well-qualified data scientist are not a dime a dozen. Instead, employees being not very familiar with data analysis are often called to do the job. Automatic machine learning can help those persons to perform predictive modeling with high performing machine learning tools without having much experience. This is achieved by making those applications parameter-free, i.e. only the data is required as input. Projects like Auto-WEKA or auto-sklearn aim to solve the Combined Algorithm Selection and Hyperparameter optimization (CASH) problem resulting in a huge optimization space. However, for most real world applications, only few different learning algorithms are required to deliver superior performances. autoxgboost simplifies this idea one step further and the CASH problem to taking Gradient Boosting as a single learning algorithm in combination with intelligent model based hyperparameter tuning. It is based on the R-Packages mlr, mlrMBO and XGBoost. It also supports categorical variables due to special inbuilt factor feature encoding. Even though autoxgboost only uses one learner instead of a whole library, it provides comparable or even better performances.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;212&quot; data-target=&quot;.212collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Awdhesh Yadav&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Household and Community factor on under-five mortality in India: An application of multilevel cox proportional hazard model&lt;/td&gt;  &lt;td&gt;multivariate, big data&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 212collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;The objective of this paper is to determine the important of community, household and individual level effect on under-five mortality in India. Using data from the latest round of Demographic Health Survey (DHS)-2005-06, multilevel cox proportional hazard analysis was performed on a nationally representative sample. The results indicate that pattern of under-five mortality were clustered within mothers and communities. The community level variables like region, place of residence, community poverty level, community education level, ethnic fractionalization index were significantly determine under-five mortality in India. The risk of under-five deaths was significantly higher for children residing in North, East and West regions compared to South region. In addition, the proportion of women in community completing secondary school were significantly more likely to increase the child survival. The household level variables like religion, caste and wealth index were significantly determining under-five mortality. The results suggest to address the contextual level factors to address under-five mortality in India&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;21&quot; data-target=&quot;.21collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Jean-Michel Perraud&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;A suite of R packages for hydrological ensemble forecasting using Rcpp&lt;/td&gt;  &lt;td&gt;space/time, performance, big data, Hydrology, forecast&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 21collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Ensemble prediction techniques have been shown to produce more accurate predictions as well as formally quantify prediction uncertainty in a range of scientific applications. We present a suite of libraries for hydrological ensemble forecasts designed for use both in research and operations. The features of the C++ libraries are available from several high-level interactive languages including R. The suite currently comprises three main R packages for rainfall forecast post processing (RPP), semi-distributed ensemble hydrological modelling (SWIFT2) and multi-dimensional ensemble time series (uchronia). The packages are designed to offer concise commands for handling ensemble time series, input/output, model parameterisation and simulation execution. The native libraries purposely have a C API for maximising interoperability and foster a consistent use experience across high-level languages. Rcpp is used for surfacing the features in the R packages. Bespoke code for marshaling data, object lifetime management and generating glue code for Rcpp is already open source and suitable for reuse in similar technical contexts.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;67&quot; data-target=&quot;.67collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Susanna Cramb&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Bayesian disease mapping in R&lt;/td&gt;  &lt;td&gt;models, space/time&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 67collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Bayesian methods prominently feature in disease mapping, and R has multiple packages designed to enable efficient computation of assorted Bayesian spatial models.Here we examine seven Bayesian models suitable for disease mapping and implement them using the R packages of R-INLA, CARBayes, R2WinBUGS and R2jags. Models considered included common approaches such as the BYM which smooth estimates over all adjacent areas, through to more recently introduced models that allowed for discontinuities between adjacent areas, as well as spline models. Simulated incidence data designed to represent a rare cancer (liver) and more common cancer (lung) were examined across 2153 areas in Australia. Model performance was compared on goodness of fit measures (WAIC, Moran’s I on residuals), computational time and convergence (Geweke). The packages themselves are also compared in terms of computational time and model flexibility. It is useful to consider several different models to understand the robustness of results when disease mapping. R has the capacity to enable a wide range of models to be considered, with the additional advantages of high quality visualisation of results.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;196&quot; data-target=&quot;.196collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Rosemary Putler&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;Analysis of EHR Data and Circulating Inflammatory Mediators: Association with Severe Clostridium difficile Infection&lt;/td&gt;  &lt;td&gt;models, applications, bioinformatics&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 196collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;Clostridium difficile infection (CDI) is a major healthcare-associated infection, and severe CDI often leads to subsequent recurrence or death. We hypothesized that circulating inflammatory mediators would associate with severity in a prospective cohort of inpatients diagnosed with CDI. An inflammatory mediator panel was performed on collected sera and merged with electronic health record (EHR) data. With these data we show that circulating biomarkers associate not only with severity of the CDI episode, but also with subsequent mortality. Because of the large number of potentially correlated predictors, our data presented a challenge when we set out to identify features to incorporate into an accurate predictive model. We explore the steps performed in this analysis, discussing methodology and decision points, including the management and analysis of EHR data, utilization of dimensional reduction techniques, and use of existing packages such as vegan, glmnet, and pROC. Through this analysis, we demonstrate how a diverse array of R packages and statistical methodologies, which function in a wide array of use cases, can also be used to answer a complex disease-related question.&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;clickable&quot; data-toggle=&quot;collapse&quot; id=&quot;200&quot; data-target=&quot;.200collapsed&quot;&gt;  &lt;td&gt;12:00&lt;/td&gt;  &lt;td&gt;Poster&lt;/td&gt;  &lt;td&gt;Stuart Davie&lt;/td&gt;  &lt;td&gt;Foyer&lt;/td&gt;  &lt;td&gt;A data driven approach to generating and scoring B2B leads&lt;/td&gt;  &lt;td&gt;visualisation, models, applications&lt;/td&gt;  &lt;td&gt;-&lt;/td&gt;&lt;/tr&gt;&lt;tr class=&quot;collapse out budgets 200collapsed&quot;&gt;  &lt;td colspan=&quot;7&quot;&gt;In many industries, companies rely on a sales team to source and qualify leads. Unfortunately, this limits a company's potential leads to those that can be manually processed, while lead qualification is limited by the quality of ad hoc scoring systems. To find leads faster, companies might engage in cold-calling, or blanket email campaigns, both of which are known for their low conversion rates. Here, a data-driven B2B lead generation and qualification solution for the UK market is presented, based on open source data and XGBoost. Our models take into account both general and company specific features, and allow an approximation of the size of market opportunity. Lead reports containing pertinent conversion information are automatically generated using xgboostExplainer and R Markdown.  Considerations on feature engineering, and difficulties associated with overfitting, are also discussed. **As there are several components to this presentation, a lightning talk would be preferred over a poster**&lt;/td&gt;&lt;/tr&gt;
</code></pre>
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->

                        </div>

                    </div>
                    <div class="col-md-3 hidden-xs hidden-sm">
                      <ul class="nav nav-pills nav-stacked">
                        <nav id="TableOfContents"></nav>
                      </ul>
                    </div>

                </div>
                

            </div>
            

            
        </div>
        

        <footer id="footer">
    <div class="container">

        
        <div class="col-md-4 col-sm-6">
            <h4>Local organising committee</h4>

            <!-- raw HTML omitted -->

            <hr class="hidden-md hidden-lg hidden-sm">

        </div>
        
        

        <div class="col-md-4 col-sm-6">

             
            <h4>Recent news</h4>

            <div class="blog-entries">
                
                
                
                
                
                
                
                
                
                    
                <h5><a href="https://user2018.r-project.org/blog/2018/07/10/wifi/">Conference wifi</a></h5>
                    
                
                
                
                
                
                
                
                
                
                
                    
                <h5><a href="https://user2018.r-project.org/blog/">News</a></h5>
                    
                
                
                
                
                
                
                
                
                
                
                    
                <h5><a href="https://user2018.r-project.org/blog/2018/06/29/lastminute/">Last minute details</a></h5>
                    
                
                
                
                
                
                
                
                
                
                
                    
                <h5><a href="https://user2018.r-project.org/blog/2018/05/30/interview-with-steph-de-silva/">Interview with Steph de Silva</a></h5>
                    
                
                
                
                
                
                
                
                
                
                
                    
                <h5><a href="https://user2018.r-project.org/blog/2018/05/17/updates/">Updates and news</a></h5>
                    
                
                
            </div>

            <hr class="hidden-md hidden-lg">
             

        </div>
        

        
        <div class="col-md-4 col-sm-6">

          <h4>Venue</h4>
          

            <!-- raw HTML omitted -->

            <a href="/contact" class="btn btn-small btn-template-main">Go to contact page</a>

            <hr class="hidden-md hidden-lg hidden-sm">

        </div>
        
        

    </div>
    
</footer>







<div id="copyright">
    <div class="container">
        <div class="col-md-4 col-sm-6">
        
          
          <p class="pull-left"><p>Copyright &copy; 2017-2018, <strong><a href='https://www.r-project.org/foundation'>R Foundation</a></strong></p></p>
            
        </div>
        <div class="col-md-4 col-sm-6">
          <p class="pull-left">
          <a href="https://twitter.com/useR2018_conf">
            <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
          </a>
          <a href="https://twitter.com/useR2018_conf">@useR2018_conf</a>
           //
          <a href="https://twitter.com/hashtag/useR2018?src=hash">#useR2018</a>
          </p>
        </div>
        <div class="col-md-4 col-sm-6">
            Template by <a href="http://bootstrapious.com/free-templates">Bootstrapious</a>.
            

            Ported to Hugo by <a href="https://github.com/devcows/hugo-universal-theme">DevCows</a>
        </div>
    </div>
</div>





    </div>
    

    
<script src="//code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>

<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0/jquery.counterup.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-parallax/1.1.3/jquery-parallax.js"></script>

<script src="//maps.googleapis.com/maps/api/js?v=3.exp"></script>

<script src="https://user2018.r-project.org/js/hpneo.gmaps.js"></script>
<script src="https://user2018.r-project.org/js/gmaps.init.js"></script>
<script src="https://user2018.r-project.org/js/front.js"></script>


<script src="https://user2018.r-project.org/js/owl.carousel.min.js"></script>


  </body>
</html>
